packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/delete-encounter-confirmation-modal.scss

```diff
@@ -0,0 +1,2 @@
+@use '@carbon/styles/scss/spacing';
+@import "../../../root.scss"
```

Comment:
I guess you can directly import the styles from root.scss, instead of creating a new file, doing the same.
Thanks!